bank-sdk/example-app/src/main/java/net/gini/android/bank/sdk/exampleapp/ui/ExtractionsActivity.kt

```diff
@@ -98,23 +106,29 @@ class ExtractionsActivity : AppCompatActivity(), ExtractionsAdapter.ExtractionsA
             editableSpecificExtractions.forEach {
                 if (!mExtractions.containsKey(it.key)) {
                     mExtractions[it.key] = GiniCaptureSpecificExtraction(
-                            it.key, "",
-                            it.value, null, emptyList())
+                        it.key, "", it.value, null, emptyList()
+                    )
                 }
             }
 
-            adapter = ExtractionsAdapter(getSortedExtractions(mExtractions), this@ExtractionsActivity, editableSpecificExtractions.keys.toList()).also {
+            adapter = ExtractionsAdapter(
+                getSortedExtractions(mExtractions),
+                this@ExtractionsActivity,
+                editableSpecificExtractions.keys.toList()
+            ).also {
                 mExtractionsAdapter = it
             }
             setOnTouchListener { _, _ ->
                 performClick()
-                val inputMethodManager = getSystemService(Activity.INPUT_METHOD_SERVICE) as InputMethodManager
+                val inputMethodManager =
+                    getSystemService(Activity.INPUT_METHOD_SERVICE) as InputMethodManager
                 inputMethodManager.hideSoftInputFromWindow(windowToken, 0)
             }
         }
     }
 
-    private fun <T> getSortedExtractions(extractions: Map<String, T>): List<T> = extractions.toSortedMap().values.toList()
+    private fun <T> getSortedExtractions(extractions: Map<String, T>): List<T> =
+        extractions.toSortedMap().values.toList()
 
     private fun sendFeedbackAndClose(binding: ActivityExtractionsBinding) {
         // Feedback should be sent only for the user visible fields. Non-visible fields should be filtered out.
```

Comment:
[Request] Rename to feedback summary:
```suggestion
    private fun sendTransferSummaryAndClose(binding: ActivityExtractionsBinding) {
        // Transfer summary should be sent only for the user visible fields. Non-visible fields should be filtered out.
```